### PR TITLE
Suggest conda --help instead of non-existing 'conda help'

### PR DESCRIPTION
**Pull request by jaimergp**
### Description

Fix https://github.com/conda/conda/issues/11585

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](https://github.com/conda/conda/blob/master/news/TEMPLATE)) for the next release's release notes?
     
- [ ] Add / update necessary tests?
- [x] Add / update outdated documentation?

### CODE REVIEW

**Comment by awwad** [comment]
Thanks for doing this!

**Comment by awwad** [comment]
Added news item.  The documentation should automatically update due to the Sphinx configuration.

File: news/11625-correct-conda-help-err-msg [hunk 1]
```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* <news item>
+
+### Bug fixes
+
+* Correct a misleading `conda --help` error message.
```
Comment (kenodegard) [inline news/11625-correct-conda-help-err-msg:7]:
```suggestion
* Correct a misleading `conda --help` error message. (#11625)
```

**Review by awwad — APPROVED** [review]
Works.

**Review by jezdez — APPROVED** [review]

**Pull request merged**